config/init.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/taskcluster/taskcluster-cli/client"
+)
+
+var (
+	Configuration map[string]map[string]interface{}
+	ConfigOptions = make(map[string]map[string]ConfigOption)
+	Credentials   *client.Credentials
+)
+
+// Setup: call it from main
```

Comment:
nit: the first line should be formatted as a complete sentence describing the function.